_posts/2024-07-10-trino-summit-2024-call-for-speakers.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Announcing Trino Summit 2024"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2023/summit-logo.png
```

Comment:
we should make a copy of the image into a new 2024 folder ... maybe over time we can make a little bit of a change so its not completely the same

Reply:
Updated link as well now